src/jit/emitarm64.cpp

```diff
@@ -6675,19 +6745,22 @@ void                emitter::emitIns_J(instruction   ins,
     }
 #endif // DEBUG
 
-    /* Assume the jump will be long */
-
-    // TODO-ARM64-Cleanup: there is only one size jump on ARM64. Clean this up.
-    id->idjShort = 1;
     if (dst != nullptr)
     {
         id->idAddr()->iiaBBlabel = dst;
-        id->idjKeepLong = emitComp->fgInDifferentRegions(emitComp->compCurBB, dst);
+
+        // Skip unconditional jump that has a single form.
+        // NYI. In case fgInDifferentRegions is true (hot/cold separation),
```

Comment:
Proper format for an NY comment is:

 // TODO-ARM64-NYI: enable hot/cold splitting